src/nvim/lua/vim.lua

```diff
@@ -118,6 +118,19 @@ local function _update_package_paths()
   last_nvim_paths = cur_nvim_paths
 end
 
+--@param s String The string to split
+--@param sep String The separator to use
+--@param plain Boolean If `true`, use the separator literally
+---(passed as an argument to String.find)
+--@returns An iterator over the split components
+---
+--- Split a string by a given separator.
```

Comment:
Can the main description go at the top? Or this is a a limitation of lua2dox ?

Reply:
I don't think it's an issue of lua2dox, but rather I can't make doxygen show up stuff the way I want. Check out the latest 2 commits.